app/client/tests/unit/DropZone.spec.ts

```diff
@@ -0,0 +1,50 @@
+import DropZone from '@/components/DropZone.vue';
```

Comment:
Here's an approach using mocking of vue3-dropzone that would allow you to test that useDropzone had been called with the expected parameters, and that the onDrop method calls the expected action (with expected file param). Can also mock the get props methods in the return value of useDropzone so can check those are use correctly in the template:
```
// Mock the import of vue3-dropzone so we can mock its useDropzone method
// We need to do this before we import DropZone
const mockGetRootProps = jest.fn().mockReturnValue({ rprop: 'rval' });
const mockGetInputProps = jest.fn().mockReturnValue({ iprop: 'ival' });
const mockUseDropzone = jest.fn().mockReturnValue({
  getRootProps: mockGetRootProps,
  getInputProps: mockGetInputProps,
});
jest.mock('vue3-dropzone', () => ({
  useDropzone: mockUseDropzone,
}));

/* eslint-disable import/first */
import DropZone from '@/components/DropZone.vue';
import { mount } from '@vue/test-utils';
import { RootState } from '@/store/state';
import Vuex from 'vuex';
import { mockRootState } from '../mocks';

describe('Dropzone', () => {
  const processFiles = jest.fn();

  const store = new Vuex.Store<RootState>({
    state: mockRootState({
      user: {
        status: 'success',
        errors: [],
        data: {
          name: 'Jane',
          id: '543653d45',
          provider: 'google',
        },
      },
      results: {
        perIsolate: {},
      },
      uploadedFiles: 0,
    }),
    actions: {
      processFiles,
    },
  });
  const wrapper = mount(DropZone, {
    global: {
      plugins: [store],
    },
  });

  test('does a wrapper exist', () => {
    expect(wrapper.exists()).toBe(true);
  });

  it('processes dropped files', () => {
    const file = {
      name: 'sample.fa',
      text: () => Promise.resolve('ACGTGTAGTCTGACGTAAC'),
    };

    expect(mockUseDropzone).toHaveBeenCalledTimes(1);
    const { onDrop, accept } = mockUseDropzone.mock.calls[0][0];
    expect(accept).toStrictEqual(['.fa', '.fasta']);
    onDrop([file]);
    expect(processFiles).toHaveBeenCalledTimes(1);
    expect(processFiles.mock.calls[0][1]).toStrictEqual([file]);
  });

  it('sets dropzone props', () => {
    expect(wrapper.find('.dropzone').attributes('rprop')).toBe('rval');
    expect(wrapper.find('input').attributes('iprop')).toBe('ival');
  });
});
```